api/crypto/frame_crypto_transformer.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2022 LiveKit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef WEBRTC_FRAME_CRYPTOR_TRANSFORMER_H_
+#define WEBRTC_FRAME_CRYPTOR_TRANSFORMER_H_
+
+#include "api/frame_transformer_interface.h"
+#include "rtc_base/buffer.h"
+#include "rtc_base/synchronization/mutex.h"
+
+namespace webrtc {
+
+class KeyManager : public rtc::RefCountInterface {
+ public:
+  enum { kMaxKeySize = 32 };
+
+ public:
+  virtual const std::vector<std::vector<uint8_t>> keys(
+      const std::string participant_id) const = 0;
+
+ protected:
+  virtual ~KeyManager() {}
+};
+
+class FrameCryptorTransformer
```

Comment:
Maybe a small detail for now, can you export the class?
So I will be able to use it when compiling a static library

Reply:
sure, I will do it.

Reply:
https://github.com/webrtc-sdk/webrtc/pull/54/commits/9f1ebc27ee83dcb4a08026c5ea47e0be21377687